paper/paper.md

```diff
@@ -176,16 +176,16 @@ In the future, the interface could be easily expanded if specific use cases requ
 
 In scientific computing, linear functions such as, e.g., spherical harmonic transforms are widespread.
 If the function $f$ is linear, differentiation becomes trivial.
-Specifically for a linear function $f$, the pushforward respectively the `jvp` of $f$ is identical to $f$ itself and independent of the position at which it is computed.
+Specifically for a linear function $f$, the pushforward or `jvp` of $f$ is identical to $f$ itself and independent of the position at which it is computed.
 Expressed in formulas, $\partial f(x)(dx) = f(dx)$ if $f$ is linear in $x$.
-Analogously, the pullback respectively the `vjp` becomes independent of the initial position and is given by the linear transpose of $f$, thus $(\partial f(x))^{T}(dy) = f^T(dy)$.
+Analogously, the pullback or `vjp` becomes independent of the initial position and is given by the linear transpose of $f$, thus $(\partial f(x))^{T}(dy) = f^T(dy)$.
 Also, all higher order derivatives can be expressed in terms of $f$ and its transpose.
 To make use of these simplifications, `JAXbind` provides a special interface for linear functions, supporting higher order derivatives, only requiring an implementation of the function and its transpose.
 
 # Platforms
 
 Currently, `JAXbind` only supports primitives that act on CPU memory.
-In the future, GPU support could be added analogous to the CPU backend.
```

Comment:
How about "In the future, GPU support could be added, which should work analogously to the CPU support in most respects."

The point here is that the effort to get things working on GPU should be much less than it was to produce the initial CPU implementation; it's just that GPU-specific knowledge is required which we don't have.

Reply:
Sounds also good to me!